qt/Application.cc

```diff
@@ -294,7 +294,7 @@ Application::Application(int& argc, char** argv) :
     myWatchDir = new WatchDir(*myModel);
 
     connect(myModel, &TorrentModel::torrentsAdded, this, &Application::onTorrentsAdded);
-    connect(myModel, &TorrentModel::torrentsChanged, myWindow, &MainWindow::refreshActionSensitivity);
+    connect(myModel, &TorrentModel::torrentsChanged, myWindow, &MainWindow::refreshActionSensitivitySoon);
```

Comment:
This looks like an implementation detail. Instead of connecting here, maybe move it to `MainWindow`'s constructor (given that we're already passing `TorrentModel` instance there). Will also allow us to make the `refreshActionSensitivitySoon` slot private, or even use lambda as we did before.

Reply:
Good call. Changed c6f07af13